feature/config.js

```diff
@@ -0,0 +1 @@
+I am a config file.
```

Comment:
Don't you know how to code?

Reply:
![images (1)](https://github.com/user-attachments/assets/18123d41-5bc9-4511-889b-c25ed4cf0d1c)